lib/core/services/auth-guard-sso-role.service.spec.ts

```diff
@@ -185,4 +190,60 @@ describe('Auth Guard SSO role service', () => {
         expect(await authGuard.canActivate(route)).toBeFalsy();
         expect(materialDialog.closeAll).toHaveBeenCalled();
     });
+
+    describe('ACS Admin', () => {
+
+        afterEach(() => {
+            authGuard.hasAcsAdminRole = undefined;
+        });
+
+        it('Should canActivate be true when the role is set to ACS_ADMIN and the user is an ACS admin', async () => {
```

Comment:
should give access to a content section (ACS_XXX) if the user has the content capability?